clipboard/dockit-clipboard.define.js

```diff
@@ -1,3 +1,3 @@
-import { Clipboard } from './src/Clipboard.js';
+import { Clipboard } from './src/Clipboard';
```

Comment:
this is gonna break Custom Element Manifest generator, it requires extension, so please add an extension everywhere where you removed it

Reply:
Is the extension mandatory in `index` files also?

Reply:
it's mandatory to track how CE classes are constructed and imported until they get defined in the Custom Element registry, including all parent classes if any
so if there is index or any other file in between and you omit extension, then it's gonna break the analyzer and it's not gonna find where the class or one of it parent classes are coming from
so as a rule of thumb: use extension, it's not that hard